glean-core/android/src/main/java/mozilla/telemetry/glean/config/Configuration.kt

```diff
@@ -37,6 +38,8 @@ internal class FfiConfiguration(
     public var uploadEnabled: Byte = uploadEnabled.toByte()
     @JvmField
     public var maxEvents: IntByReference = if (maxEvents == null) IntByReference() else IntByReference(maxEvents)
+    @JvmField
+    public var deferCollection: Byte = deferCollection.toByte()
```

Comment:
This is not being exposed nor documented in the `Configuration` object below. Is this intentional?

Reply:
Yes, it's intentional because, as I understand it, this property will always be `false` for Kotlin and Swift, since this is something we are adding for desktop. Do you think it would be useful to make it configurable for mobile platforms?

Reply:
I think it's worth adding it here as well or, at the very least, document why it's not here.